sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -2232,13 +2232,22 @@ class Analyzer(override val catalogManager: CatalogManager) extends RuleExecutor
             }
           }
 
-          case u @ UnresolvedFunction(nameParts, arguments, _, _, _) => withPosition(u) {
-            resolveBuiltinOrTempFunction(nameParts, arguments, Some(u)).getOrElse {
+          case u @ UnresolvedFunction(nameParts, arguments, _, _, _, sortOrder) => withPosition(u) {
+            val args = sortOrder match {
+              case Some(s) if nameParts.length == 1 &&
+                (nameParts.head == "percentile_cont" || nameParts.head == "percentile_disc") =>
```

Comment:
can we lookup the function first then check the expression? it's hacky to check the name here.

Reply:
Yes. These code just is a temp implementation. It's used let you give a first review.